ams/server/src/main/java/com/netease/arctic/server/optimizing/OptimizingQueue.java

```diff
@@ -123,6 +124,18 @@ public void removeOptimizer(String resourceId) {
     authOptimizers.values().removeIf(op -> op.getResourceId().equals(resourceId));
```

Comment:
It was not removed from sysdb.

I think we need a method to encapsulate deleting the optimizer from sysdb and deleting from memory(`authOptimizers`).

And also, the removed optimizer should also be removed from `optimizingQueueByToken` in `DefaultOptimizingService`, to avoid memory leaks.